Core/ContentBlockerConfigurationUserDefaults.swift

```diff
@@ -71,13 +72,17 @@ public class ContentBlockerConfigurationUserDefaults: ContentBlockerConfiguratio
         var whitelist = domainWhitelist
         whitelist.insert(domain)
         domainWhitelist = whitelist
-        SFContentBlockerManager.reloadContentBlocker()
+        onStoreChanged()
     }
     
     public func removeFromWhitelist(domain: String) {
         var whitelist = domainWhitelist
         whitelist.remove(domain)
         domainWhitelist = whitelist
-        SFContentBlockerManager.reloadContentBlocker()
+        onStoreChanged()
+    }
+    
+    private func onStoreChanged() {
```

Comment:
`notifiyStoreChanged` might be a better name for this?  `onXXX` is usually the method name in the receiver of such events.

Reply:
Actually, can see a couple of places where this convention is used - feel free to ignore.